app/(afterLogin)/(incentive)/incentives/compensation/all/table/index.tsx

```
@@ -93,55 +93,55 @@ const AllIncentiveTable: React.FC = () => {
-            id: item?.id,
-            userId: item?.userId,
-            recognition: item?.recognitionType || '--',
-            employee_name: (
-              <Tooltip>
-                <div className="flex flex-wrap items-center justify-start gap-3">
-                  <Avatar icon={<UserOutlined />} />
-                  <span>
-                    {getEmployeeInformation(item?.userId)?.firstName +
-                      '  ' +
-                      getEmployeeInformation(item?.userId)?.middleName}
+        return {
```

ESLint

[error] Insert `··`

(prettier/prettier)